src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -898,12 +898,14 @@ private void CheckFeatureAvailability(MessageID feature)
                 return;
             }
 
-            if (feature.RequiredFeature() != null)
+            var requiredFeature = feature.RequiredFeature();
+            if (requiredFeature != null)
             {
                 if (!options.IsFeatureEnabled(feature))
                 {
-                    this.AddError(ErrorCode.ERR_FeatureIsExperimental, feature.Localize());
+                    this.AddError(ErrorCode.ERR_FeatureIsExperimental, feature.Localize(), requiredFeature);
```

Comment:
It looks like there is one more call site with similar issue - Binder_Symbols.cs(1956)